src/hardware/mixer.cpp

```diff
@@ -260,11 +304,17 @@ static void configure_reverb(std::string reverb_pref)
 	if (reverb_pref == "on")
 		reverb_pref = "medium";
 
-	enum class RoomSize { tiny, small, medium, large, huge };
-	const auto room_size = enum_cast<RoomSize>(reverb_pref).value();
-
 	auto &r = mixer.reverb; // short-hand reference
 
+	auto current_preset = r.preset;
+	auto new_preset = enum_cast<ReverbPreset>(reverb_pref).value();
+	auto preset_changed = (current_preset == new_preset);
```

Comment:
Probably meant `preset_unchanged`

Reply:
Yeah, I remember I flipped the condition at some point :)